test/apiv2/10-images.at

```diff
@@ -196,12 +196,16 @@ fi
 curl -XPOST --data-binary @<(cat $CONTAINERFILE_TAR) \
     -H "content-type: application/tar" \
     --dump-header "${TMPD}/headers.txt" \
-    -o /dev/null \
-    "http://$HOST:$PORT/v1.40/libpod/build?dockerfile=containerfile" &> /dev/null
+    -o "${TMPD}/response.txt" \
+    "http://$HOST:$PORT/v1.40/libpod/build?dockerfile=containerfile&q=true" &> /dev/null
```

Comment:
This checks the libpod endpoint however the extra output was only printed in the compat endpoint